src/lib.rs

```diff
@@ -194,9 +194,13 @@ use crate::ffi::CPtr;
 pub use crate::key::{InvalidParityValue, Keypair, Parity, PublicKey, SecretKey, XOnlyPublicKey};
 pub use crate::scalar::Scalar;
 
-/// Trait describing something that promises to be a 32-byte random number; in particular,
-/// it has negligible probability of being zero or overflowing the group order. Such objects
-/// may be converted to `Message`s without any error paths.
+/// Trait describing something that promises to be a 32-byte uniformly random number.
+///
+/// In particular, anything implementing this trait must have neglibile probability
+/// of being zero, overflowing the group order, or equalling any specific value.
+///
+/// Since version 0.29 this has been deprecated; users should instead implement
+/// `Into<Message>` for types that satisfy these properties.
```

Comment:
I wonder when are we going to remove this entirely.

Reply:
It was deprecated in April. Let's give it at least a year (unless it winds up blocking 1.0). I'd give it 2. It's self-contained and not hurting anything.